content/builds/titan/inmost_light.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Heart of Inmost Light"
+description: "Empower your abilities and use them more frequently in any activity."
+season: 15
+draft: false
+tags: ["titan", "pve", "pvp", "heart of inmost light", "ability spam", "class-neutral"]
+---
+
+
+## Overview
+This build allows you to power up your abilities and speed up their cooldown. With the right subclass, it's even viable in GMs.
+
+### Subclass: 
+This works with any subclass but there are two that offer more viability in tougher activities like Grandmaster ordeals and Master lost sectors:
+* [Code of the Seigebreaker](https://d2.destinygamewiki.com/wiki/Code_of_the_Siegebreaker) 
+  * This one excels in add-clear situations.
+* [Code of the Commander](https://d2.destinygamewiki.com/wiki/Code_of_the_Commander) 
+  * This is good for adclear too but its detonators and super make it more useful for encounters with large enemy vehicles and
+
+
+### Stat Requirements
+Discipline will be the main focus but a good baseline of strength and resilience are nice to help keep abilities empowered on a steady rotation. Like everything, you'll still want some recovery as well.
+``` 
+Mobility   : -
+Resilience : 50+
+Recovery   : 80+
+Discipline : 90+
+Intellect  : -
+Strength   : 50+
+```
+* A "`-`" means the stat isn't a priority for the build.
+
+### Armor
+[Heart of Inmost Light](https://www.light.gg/db/items/1341951177/heart-of-inmost-light/) will be the keystone of this build  This exotic is sub-class-neutral too, so you can use it with your preferred sub-class.
+
+### Armor Mods
+[Heart of Inmost Light](https://www.light.gg/db/items/1341951177/heart-of-inmost-light/) is strong enough on its own but there are some potent mod combinations that can push things even further:
+* Kickstart mods:
+  * [Grenade Kickstart](https://www.light.gg/db/items/47859594/grenade-kickstart/) (stasis)
+  * [Melee Kickstart](https://www.light.gg/db/items/1371877916/melee-kickstart/) (stasis)
+  * [Utility Kickstart](https://www.light.gg/db/items/445559589/utility-kickstart/) (stasis)
```

Comment:
same as my other comment.  it might be good to mention you have to have stasis powered gauntlets/cloak here.